astroquery/mast/missions.py

```diff
@@ -0,0 +1,187 @@
+class MastMissionsClass(MastQueryWithLogin):
+    """
+    MastMissions search class.
+
+    Class that allows direct programatic access to the MAST search API for a given mission.
+    """
+
+    def __init__(self, *, mission='hst', service='search'):
+        super().__init__()
+
+        self._search_option_fields = ['limit', 'offset', 'sort_by', 'search_key', 'sort_desc', 'select_cols',
+                                      'skip_count', 'user_fields']
+        self.service = service
+        self.mission = mission
+
+        service_dict = {self.service: {'path': self.service, 'args': {}}}
+        self._service_api_connection.set_service_params(service_dict, f"{self.service}/{self.mission}")
+
+    def _parse_result(self, response, verbose=False):  # Used by the async_to_sync decorator functionality
+        """
+        Parse the results of a `~requests.Response` objects and return an `~astropy.table.Table` of results.
+
+        Parameters
+        ----------
+        response : `~requests.Response`
+            `~requests.Response` objects.
+        verbose : bool
+            (presently does nothing - there is no output with verbose set to
+            True or False)
+            Default False.  Setting to True provides more extensive output.
+
+        Returns
+        -------
+        response : `~astropy.table.Table`
+        """
+
+        return self._service_api_connection._parse_result(response, verbose, data_key='results')
+
+    @class_or_instance
+    def query_region_async(self, coordinates, radius=3*u.arcmin, **kwargs):
+        """
+        Given a sky position and radius, returns a list of matching dataset IDs.
+
+        Parameters
+        ----------
+        coordinates : str or `~astropy.coordinates` object
+            The target around which to search. It may be specified as a
+            string or as the appropriate `~astropy.coordinates` object.
+        radius : str or `~astropy.units.Quantity` object, optional
+            Default 3 degrees.
+            The string must be parsable by `~astropy.coordinates.Angle`. The
+            appropriate `~astropy.units.Quantity` object from
+            `~astropy.units` may also be used. Defaults to 0.2 deg.
+        **kwargs
+            Other mission-specific keyword args.
+            These can be found at the following link
+            https://mast.stsci.edu/search/docs/#/Hubble%20Search/post_search_hst_api_v0_1_search_post
+            For example one can specify the output columns(select_cols) or use other filters(conditions)
+
+        Returns
+        -------
+        response : list of `~requests.Response`
+        """
+
+        # Put coordinates and radius into consistant format
+        coordinates = commons.parse_coordinates(coordinates)
+
+        # if radius is just a number we assume degrees
+        radius = coord.Angle(radius, u.arcmin)
+
+        # basic params
+        params = {'target': [f"{coordinates.ra.deg} {coordinates.dec.deg}"],
```

Comment:
Does it throw off the script if the coordinates are not separated by `,` ?

Reply:
No, It won't. It takes in both a two word string and a list of comma separated values.